src/clj/game/cards/assets.clj

```diff
@@ -988,8 +988,7 @@
    "PAD Factory"
    {:abilities [{:cost [:click 1]
                  :label "Place 1 advancement token on a card"
-                 :choices {:req #(and (= (:side %) "Corp")
-                                      (installed? %))}
+                 :choices {:req #(installed? %)}
```

Comment:
This works fine--it can also be as simple as `{:req installed?}`

Reply:
Cool, I assumed it has to be the whole anon function. I've simplified it in the latest commit.